pkg/filterengine/global.go

```diff
@@ -0,0 +1,25 @@
+package filterengine
+
+import "github.com/infracloudio/botkube/pkg/filterengine/filters"
+
+var (
+	// DefaultFilterEngine contains default implementation for FilterEngine
+	DefaultFilterEngine FilterEngine
+)
+
+// SetupGlobal set ups a global filter engine with all default filters enabled
+// TODO: Convert it to local instance shared by all BotKube components
+// 	See: https://github.com/infracloudio/botkube/issues/589
+func SetupGlobal() {
+	filtersToRegister := []Filter{
+		filters.ImageTagChecker{Description: "Checks and adds recommendation if 'latest' image tag is used for container image."},
+		filters.IngressValidator{Description: "Checks if services and tls secrets used in ingress specs are available."},
+		filters.ObjectAnnotationChecker{Description: "Checks if annotations botkube.io/* present in object specs and filters them."},
+		filters.PodLabelChecker{Description: "Checks and adds recommendations if labels are missing in the pod specs."},
+		filters.NamespaceChecker{Description: "Checks if event belongs to blocklisted namespaces and filter them."},
+		filters.NodeEventsChecker{Description: "Sends notifications on node level critical events."},
+	}
+
+	DefaultFilterEngine = NewDefaultFilter()
+	DefaultFilterEngine.Register(filtersToRegister...)
```

Comment:
Hey @pkosiec,
Any specific reason behind changing the filter registration workflow? The goal was to make it as easy as possible to develop and register new filters for users. Do you find any drawbacks to the current approach?
https://www.botkube.io/filters/

If we decided to change this, we might want to update https://www.botkube.io/filters/

Reply:
Yes, this is something I responded here: https://github.com/infracloudio/botkube/pull/593#discussion_r882289487